src/FreeAssAlgebra.jl

```diff
@@ -174,8 +174,8 @@ end
 
 function FreeAssociativeAlgebra(
    R::AbstractAlgebra.Ring,
-   s::Vector{T};
-   cached::Bool = true) where T <: Union{String, Char}
+   s::Union{AbstractVector{<:AbstractString}, AbstractVector{Symbol}, AbstractVector{Char}};
+   cached::Bool = true)
```

Comment:
with the non-copying constructor for Vector{Symbol}, I was always surprised no one complained about the following behaviour.
```julia
julia> vars = [:a, :b, :c];

julia> R, (a, b, c) = FreeAssociativeAlgebra(QQ, vars);

julia> a+b+c
a + b + c

julia> vars[2] = :notb
:notb

julia> a+b+c
a + notb + c
```

Reply:
We don't do this anywhere else I believe. We should probably not start it here.

Soon immutable arrays we land in julia, which we can use once 1.8/1.9 is out :)